server/migrations/20181107082847-create-rating.js

```diff
@@ -0,0 +1,41 @@
+export default {
+  up: (queryInterface, Sequelize) => queryInterface.createTable('Ratings', {
+    id: {
+      allowNull: false,
+      autoIncrement: true,
+      primaryKey: true,
+      type: Sequelize.INTEGER
+    },
+    userId: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      allowNull: false,
+      references: {
+        model: 'Users',
+        key: 'id'
+      }
+    },
+    articleId: {
+      type: Sequelize.INTEGER,
+      onDelete: 'CASCADE',
+      allowNull: false,
+      references: {
+        model: 'Articles',
+        key: 'id'
+      }
+    },
+    rating: {
+      type: Sequelize.INTEGER,
+      allowNull: false
```

Comment:
If default value is null, why is allowNull false? Please check through the entire PR and make sure this and other things are corrected.

Reply:
I am looking at this now.

Reply:
This is fixed now.